common/gratia/common/sandbox_mgmt.py

```diff
@@ -582,7 +585,9 @@ def OpenNewRecordFile(dirIndex):
                 f = open(filename, 'w')
                 dirIndex = index
                 return (f, dirIndex)
-            except:
+            except Exception as e:
+                DebugPrint(0, 'Caught exception while creating file: ', e)
+                DebugPrintTraceback()
```

Comment:
```suggestion
                DebugPrintTraceback(0)
```

Can't we just use this instead of the DebugPrint above? From https://github.com/opensciencegrid/gratia-probe/blob/2.x/common/gratia/common/debug.py#L162-L166

Reply:
Yup, that's fine.  I didn't read much into the DebugTraceback.  Though, the debug print does give some "context" to the exception.

Reply:
```suggestion
```

Good point on the additional context. Let's just get rid of it since it's superfluous.